lib/src/color/_setting.dart

```diff
@@ -33,4 +33,12 @@ mixin SketchColorSetting on BaseSketch {
   void noStroke() {
     _paintingContext.strokePaint.color = const Color(0x00000000);
   }
+
+  void clear() {
+    _backgroundColor = const Color(0x00000000);
+
+    _paintingContext.canvas.drawColor(const Color(0x00000000), BlendMode.clear);
```

Comment:
Does this need the `BlendMode.clear` specified? What happens if you don't pass a blend mode?

Reply:
Actually the `BlendMode.clear` is what matters. By applying the `BlendMode.clear`, it drops both the source and destination images, which turns the whole canvas's color to transparent. The color `Color(0x00000000)` does not matter.

I tried using `background(color: Color(0x00000000))` before, it draws a transparent rect which has no effect. So that's why I choose this way.